update for eforms, add award.buyers (#16)
open-contracting-extensions/ocds_contract_buyer_extension: README.md

> # Multiple buyers information

Reviewer: We already allow multiple buyers in plain OCDS (at the tender stage), so we need to distinguish this more clearly. I don't know a succinct way, so I'm suggesting this for now.

```suggestion
# Buyer per award or contract
```

> The core OCDS schema provides space for a single `buyer` to be described for each contracting process. The buyer is defined as the organization whose budget will be used for the purchase of the goods, services or works described in the contract.

Reviewer: ```suggestion
Adds buyer fields to the award and contract objects to indicate the buyer(s) for an individual award or contract, if different from the buyer(s) involved in the contracting process as a whole.

OCDS indicates the (lead) buyer in a contracting process with the `buyer` field. If a contracting process involves multiple buyers, each buyer can be added to the `parties` array with a 'buyer' role.
```

> However, some forms of contracting process, such as framework agreements, may result in multiple awards or contracts with a different buyer or buyers for each. This extension adds `awards.buyers` and `contracts.buyer` objects to support these scenarios.

Reviewer: FA call-offs are modelled as separate contracting processes, so I don't think this extension is needed for that case.

```suggestion
However, in some cases, for a given award, a subset of the buyers decide on the supplier and sign the contract. **Only use this extension in such cases.**
```

## Legal Context

> In the European Union, this extension's fields correspond to [eForms OPT-300-Contract-Signatory](https://docs.ted.europa.eu/eforms/latest/reference/business-terms/). See [OCDS for eForms](https://standard.open-contracting.org/profiles/eforms/latest/en/) for correspondences to eForms fields.

Reviewer: I applied the better wording to all extensions, so we can use it now :)

```suggestion
In the European Union, this extension's fields correspond to [eForms OPT-300-Contract-Signatory](https://docs.ted.europa.eu/eforms/latest/reference/business-terms/). For correspondences to eForms fields, see [OCDS for eForms](https://standard.open-contracting.org/profiles/eforms/latest/en/).
```

## Example

### Contracts

> 2 contracts are issued with a different buyer for each.

Reviewer 2: I don't think it's written down anywhere, but I think we generally write smaller numbers as words.

```suggestion
Two contracts are issued with a different buyer for each.
```

}
```

### Awards

> 1 award is issued with 2 buyers.

Reviewer 2: ```suggestion
One award is issued with two buyers.
```

```json
{
  "awards": [
    {
      "id": "1",
      "buyers": [
        {
          "name": "Example Department of Education",
          "id": "GB-GOV-12345678"
        },
        {
          "name": "Example Department of Transport",
          "id": "GB-GOV-00000000"
        }
      ]
    }
  ]
}
```

## Issues

Report issues for this extension in the [ocds-extensions repository](https://github.com/open-contracting/ocds-extensions/issues), putting the extension's name in the issue's title.

> ## Changelog

Reviewer: Move changelog before issues.

### 2023-04-12

> * Add `awards.buyers` object

Reviewer: ```suggestion
* Add `awards.buyers` field.
```
